docs/StandaloneConfiguration.md

```diff
@@ -1,8 +1,33 @@
 # RepoSense - Guide to Setup Standalone Configuration
-
+### Quick Start
 1. Add [_reposense/config.json](../_reposense/config.json) to the root of your repository.
-1. Fill it up with author information, such as the GitHub Id, Display Name (Optional) and the [Author Names](UserGuide.md#git-author-name) (Optional).
+1. Fill it up with author information, such as the GitHub Id, Display Name (Optional), [Author Names](UserGuide.md#git-author-name)  (Optional), [Ignore Glob List](UserGuide.md#csv-config-file) (Optional).
+
+### Detailed explanation of the structure of repo-config.json
+```
+{
+  "ignoreGlobList": ["**.dat", "**.js"],    <-- Repository level's list of file formats to ignore 
+  "authors": [                              
+  {                                            
+    "githubId": "alice",                       
+    "displayName": "Alice T.",
+    "authorNames": ["AT", "A"],
+    "ignoreGlobList": ["**.css"]            <-- Author level's is cumulative to Repository level's ignoreGlobList.
+  },                                            Thus, the actual ignoreGlobList for alice would contains
+  {                                             css, dat and js.
+    "githubId": "bob"                       
+                                            <-- Optional information can be left out to prevent clutter.
+  }
+ ]
+}
+```
```

Comment:
It's best not to include comments here. Reason: users can copy paste code from here to their config.json. Can give the comments separately.